packages/flutter_tools/lib/src/macos/xcode.dart

```diff
@@ -260,28 +282,7 @@ class XCDevice {
     );
   }
 
-  bool get isInstalled => _xcode.isInstalledAndMeetsVersionCheck && xcdevicePath != null;
-
-  String _xcdevicePath;
```

Comment:
This check is overkill.  If Xcode and `xcrun` is installed, `xcdevice` is installed.